Assignments/A3/Distro/QLearn.c

```diff
@@ -416,3 +440,88 @@ int pos_to_index(int pos[2], int size_X) { return pos[0] + pos[1] * size_X; }
 int is_pos_valid(int pos[2], int size_X, int size_Y) {
   return 0 <= pos[0] && pos[0] < size_X && 0 <= pos[1] && pos[1] < size_Y;
 }
+
+// Avg Cat Dist Feature
```

Comment:
Describe more the idea of this feature function - what does it do that's more than simply the "average cat distance"?